repo/packages/M/marathon/0/config.json

```diff
@@ -0,0 +1,378 @@
+        },
+        "ssl-keystore-password" : {
+          "description" : "Password for the keystore supplied with the `ssl_keystore_path` option. Required if `ssl_keystore_path` is supplied. May also be specified with the `MESOSPHERE_KEYSTORE_PASS` environment variable.",
+          "type" : "string"
+        },
+        "ssl-keystore-path" : {
+          "description" : "Path to the SSL keystore. HTTPS (SSL) will be enabled if this option is supplied. Requires `--ssl_keystore_password`. May also be specified with the `MESOSPHERE_KEYSTORE_PATH` environment variable.",
+          "type" : "string"
+        },
+        "store-cache" : {
+          "description" : "Yes: Enable an in memory cache for the storage layer.. No: Disable the in memory cache for the storage layer ",
+          "type" : "boolean",
+          "default" : true
+        },
+        "task-launch-confirm-timeout" : {
+          "description" : "Time, in milliseconds, to wait for a task to enter the TASK_STAGING state before killing it.",
+          "type" : "integer",
+          "default" : 300000
+        },
+        "task-launch-timeout" : {
+          "description" : "Time, in milliseconds, to wait for a task to enter the TASK_RUNNING state before killing it.",
+          "type" : "integer",
+          "default" : 300000
+        },
+        "tracing" : {
+          "description" : "Yes: Enable trace logging of service method calls. No: (Default) Disable trace logging of service method calls",
+          "type" : "boolean",
+          "default" : false
+        },
+        "webui-url" : {
+          "description" : "The http(s) url of the web ui, defaulting to the advertised hostname",
+          "type" : "string"
+        },
+        "zk" : {
+          "description" : "ZooKeeper URL for storing state. Format: zk://host1:port1,host2:port2,.../path",
+          "type" : "string"
+        },
+        "zk-compression" : {
+          "description" : "Yes: Enable compression of zk nodes, if the size of the node is bigger than the configured threshold.. No: Disable compression of zk nodes",
+          "type" : "boolean",
+          "default" : true
+        },
+        "zk-compression-threshold" : {
+          "description" : "Threshold in bytes, when compression is applied to the zk node (Default: 64 KB).",
+          "type" : "integer",
+          "default" : 65536
+        },
+        "zk-max-versions" : {
+          "description" : "Limit the number of versions, stored for one entity.",
+          "type" : "integer",
+          "default" : 25
+        },
+        "zk-session-timeout" : {
+          "description" : "The timeout for zookeeper sessions in milliseconds",
+          "type" : "integer",
+          "default" : 10000
+        },
+        "zk-timeout" : {
+          "description" : "The timeout for ZooKeeper in milliseconds",
+          "type" : "integer",
+          "default" : 10000
+        }
+      },
+      "required" : [ "master", "framework-name" ],
```

Comment:
We should document that default since this is hard to find in the long command string of the marathon.json.